packages/wallets/src/routes/WalletsListingRoute/WalletsListingRoute.scss

```diff
@@ -5,10 +5,10 @@
     flex-direction: column;
     align-items: center;
     justify-content: center;
-    gap: 2.4rem;
```

Comment:
Can't we keep the gap and remove the `padding-bottom` in `.wallets-list-header`, the  `margin-bottom` in `.wallets-desktop-wallets-list`?

Reply:
This was removed due to the small gap for mobile (0.8rem) but I've updated the styling to have a different gap for mobile and matched the spacings for the sections